src/main/sensors/compass.c

```diff
@@ -70,6 +70,11 @@ bool compassInit(int16_t magDeclinationFromConfig)
     return ret;
 }
 
+bool compassIsWorking(void)
```

Comment:
The check should probably be smarter than this. Please make it a generic function, we'll extend it further.